frappe/website/doctype/blog_post/blog_post.py

```diff
@@ -61,7 +61,7 @@ def get_context(self, context):
 
 
 		context.content = get_html_content_based_on_type(self, 'content', self.content_type)
-		context.description = self.blog_intro or context.content[:140]
+		context.description = self.blog_intro or strip_html_tags(context.content[:140])
```

Comment:
Any particular reason for stripping html tags here ?

Reply:
@hrwX I have mentioned it in the PR description

Reply:
@ruchamahabal Its getting duplicated because we are appending 140 characters of content to description, but here the 140 character is the iframe tag itself and hence it is being displayed twice.

Reply:
@hrwx yes, I absolutely get that. If the intro is not present in the blog post, `get_context` method fetches the first 140 characters from the content field and sets it as blog intro. Hence this is what is happening

![strip_html_issue](https://user-images.githubusercontent.com/24353136/69244903-e36c1780-0bcb-11ea-9923-95f35aab50db.gif)

Moreover, `strip_html_tags` is also used in the validate method if blog intro is not explicitly added.
```
if not self.blog_intro:
	content = get_html_content_based_on_type(self, 'content', self.content_type)
        self.blog_intro = content[:140]
        self.blog_intro = strip_html_tags(self.blog_intro)
```
As the field description says 'Description for listing page, in plain text, only a couple of lines. (max 140 characters)', I suppose there won't be HTML tags used in this field.
And if this is correct, I feel that strip_html_tag should be after the two if conditions in validate so that html tags are striped even if the intro is added explicitly.